README.md

```diff
@@ -0,0 +1,53 @@
+# restbase-mod-table-sqlite [![Build Status](https://travis-ci.org/wikimedia/restbase-mod-table-sqlite.svg?branch=master)](https://travis-ci.org/wikimedia/restbase-mod-table-sqlite)
+
+An SQLite3 back-end module for [RESTBase](wikimedia/restbase) conforming to the
+[RESTBase storage specification](wikimedia/restbase-mod-table-spec).
+
+## Installation
+
+Firstly, install RESTBase. The SQLite back-end module should be pulled in
+automatically as a dependency. If you cannot find `restbase-mod-table-sqlite` in
```

Comment:
[We already do so](https://github.com/wikimedia/restbase/blob/58d0d733fcf1bd625a20cfcdf67b9cdce5e0ca13/package.json#L58)